test/test_maxwellDriver1d.py

```diff
@@ -1057,3 +1058,64 @@ def test_buildDrivedEvolutionOperator():
     plt.spy(A_drived, markersize=4)
     # plt.spy(A, markersize=4)
     plt.show()
+
+def test_check_initial_conditions_GW_right():
```

Comment:
please change the name to test_fdtd_check... to follow the same convention as with the other tests. Also, put it next to the other fdtd tests, not at the end of file.